packages/eip1193-permission-middleware/README.md

```diff
@@ -0,0 +1,15 @@
+# `@metamask/eip1193-permission-middleware`
+
+Implements the JSON-RPC methods for managing permissions as referenced in [EIP-2255](https://eips.ethereum.org/EIPS/eip-2255) and [MIP-2](https://github.com/MetaMask/metamask-improvement-proposals/blob/main/MIPs/mip-2.md), but supporting [chain-agnostic permission caveats](https://github.com/MetaMask/chain-agnostic-permission) in alignment with [`@metamask/multichain-api-middleware`](https://github.com/MetaMask/multichain-api-middleware).
```

Comment:
Does this sound right?

Reply:
i don't think these links are correct

Reply:
Oh, this is outdated. I've updated the repo links to what they will be when published, so https://npmjs.com/package/@metamask/chain-agnostic-permission and https://npmjs.com/package/@metamask/multichain-api-middleware.